symphony-bdk-spring/symphony-bdk-core-spring-boot-starter/src/main/java/com/symphony/bdk/spring/annotation/SlashAnnotationProcessor.java

```diff
@@ -32,6 +37,8 @@
  * @see <a href="https://github.com/spring-projects/spring-framework/blob/master/spring-context/src/main/java/org/springframework/context/event/EventListenerMethodProcessor.java">EventListenerMethodProcessor.java</a>
  */
 @Slf4j
+@Generated // means excluded from test coverage: error cases hard to test,
```

Comment:
Could we move this annotation on specific methods that we can't fully cover? Assuming that most of the class can still be correctly covered